src/syslog_mcp/services/gotify_client.py

```
@@ -0,0 +1,180 @@
+"""
+Gotify client for sending alert notifications.
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)